### PR TITLE
Paranthesize complement printing #9877

### PR DESCRIPTION
**Pull request by kshitij10496**
Tried to fix the Complement printing issue #9877

### CODE REVIEW

**Comment by kshitij10496** [comment]
ping @gxyd @aktech @hargup @shivamvats

**Comment by gxyd** [comment]
You can update your previous PR, instead of making a new one.

File: sympy/printing/pretty/pretty.py [hunk 1]
```diff
@@ -2051,3 +2051,4 @@ def pager_print(expr, **settings):
     if 'num_columns' not in settings:
         settings['num_columns'] = 500000  # disable line wrap
     pager(pretty(expr, **settings).encode(getpreferredencoding()))
+
```
Comment (gxyd) [inline sympy/printing/pretty/pretty.py]: Here you introduced a new line. Hence it fails `code_quality.py` test.

File: sympy/printing/pretty/tests/test_pretty.py [hunk 2]
```diff
@@ -5176,3 +5176,9 @@ def test_issue_6134():
      0                              0                   \
 """)
     assert upretty(e) == ucode_str
+
```
Comment (gxyd) [inline sympy/printing/pretty/tests/test_pretty.py]: One more blank line.

**Comment by kshitij10496** [comment]
@gxyd I did those small changes. I squashed those commits into one. The extra commit -> https://github.com/kshitij10496/sympy/commit/dc0dcfc0bfc4763f742e1f272e2306fbca0c1a2c appearing in my PR is maybe because while interactive rebasing for squash, I included the other commit too. That's why it has been "re-committed" and hence showing up in my PR. I think I can't undo that except for making a new branch.

File: sympy/printing/pretty/tests/test_pretty.py (hunk 2 above)
Comment (kshitij10496) [inline sympy/printing/pretty/tests/test_pretty.py]: Done.

**Comment by gxyd** [comment]
Probably someone else may be able to help you, since I am not experience with this problem on git.

**Comment by kshitij10496** [comment]
@gxyd  
I am thinking on working on this issue in a separate branch.
Is this fine ?

**Comment by gxyd** [comment]
Sure you can go ahead, :)

**Pull request closed without merging**